src/components/TodoList/TodoList.tsx

```diff
@@ -1,6 +1,17 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
+type Props = {
+  todos: Todo[];
+  selectTodo: Todo | null;
```

Comment:
Consider renaming `selectTodo` to `selectedTodo` to better reflect its purpose and align with naming conventions.

Reply:
> Consider renaming `selectTodo` to `selectedTodo` to better reflect its purpose and align with naming conventions.

Good point